modules/openapi-generator/src/main/resources/JavaJaxRS/spec/api.mustache

```diff
@@ -4,11 +4,9 @@ package {{package}};
 {{/imports}}
 
 import javax.ws.rs.*;
-import javax.ws.rs.core.Response;
+{{#returnResponse}}import javax.ws.rs.core.Response;{{/returnResponse}}
```

Comment:
I think you will need to do it in postProcessOperations to loop through all operations and create something like `x-jaxrs-spec-has-response` instead. PM me via Slack (https://join.slack.com/t/openapi-generator/shared_invite/enQtNzAyNDMyOTU0OTE1LTY5ZDBiNDI5NzI5ZjQ1Y2E5OWVjMjZkYzY1ZGM2MWQ4YWFjMzcyNDY5MGI4NjQxNDBiMTlmZTc5NjY2ZTQ5MGM)